Pythagora/taks3.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# task3
+class Pythagora
+  def initialize(a, b, c)
+    @sides = [a, b, c].sort
+    raise ArgumentError if @sides.any?(&:negative?)
+  end
+
+  def pythagorean?
+    @sides[2]**2 == @sides[0]**2 + @sides[1]**2
+  end
+
+  def isoscele?
+    @sides.any? { |side| @sides.count(side) == 2 }
+  end
+
+  def equilateral?
+    @sides.all? { |side| @sides.count(side) == 3 }
+  end
+
+  def calculate
+    if pythagorean?
```

Comment:
where is else?

  if pythagorean?
      puts 'Triangle is pythagorean'
    elsif isoscele?
      puts 'Triangle is isoscele and rectangular'
    elsif equilateral?
      puts 'Triangle is equilateral but not rectangular'
    else
    
    end